### ISSUE TITLE
Demo application sticks on a mkmapview

### ISSUE THREAD

**Issue by liamwalsh** [chriswebb09/ARKitNavigationDemo#2]
Hello 
Upon launching the app I can see lots of information in the logs, but the app itself freezes on a mkmapview - you can click and drag around the map, but long pressing etc doesn't do anything.

**Comment by chriswebb09** [comment]
Hi! Thanks for getting back to me! Sorry about the glitch! It's not actually frozen, I've had the same thing happen. For some reason it has trouble registering the touches on occasion. To account for that, make sure your press is at least half a second, and press on a road. Also make sure you release after half a second. It's sort of goldilocks. You just have to try it a few times to get it to work. I'm trying to work out a fix.

**Comment by chriswebb09** [comment]
Update: So I think I fixed it. Let me know if the latest update works better.

**Comment by staz1000** [comment]
I can't get anything to work either, I'm on your latest commit.

**Comment by chriswebb09** [comment]
I'll try to figure out what's going on. In the meantime, I'll create a branch where it's easy for you to add in hard coded destinations if you want to see the AR navigation in action. Sorry for the confusion.

**Comment by jagatfx** [comment]
@chriswebb09 I don't see the code calling requestWhenInUseAuthorization(). That prevents everything else from continuing. When I added that call to `LocationService` I was able to get things working. I'd also suggest adjusting the order of the configuration, having it occur before calling the startUpdating functions at https://github.com/chriswebb09/ARKitNavigationDemo/blob/master/ARKitNavigationDemo/ARKitNavigationDemo/Services/LocationService.swift#L28
